### ISSUE TITLE
Unreal Engine 5. Live coding failed. Cannot open file RiderLink\Binaries\Win64\UnrealEditor-Rider***.patch_0.exe while linking

### ISSUE THREAD

**Issue by NullOne01** [JetBrains/UnrealLink#205]
I'm using Unreal Engine 5.0.3 from Epic store. Windows 10.

While doing live coding, all of my assets were compiling and linking successfully. However, after each live coding iteration there is an error, which I was ignoring:

```
Building patch from 16 file(s) for Live coding module F:\Projects\Unreal\CryptRaider\CryptRaider\Binaries\Win64\UnrealEditor-CryptRaider.dll
   Creating library F:\Projects\Unreal\CryptRaider\CryptRaider\Binaries\Win64\UnrealEditor-CryptRaider.patch_0.lib and object F:\Projects\Unreal\CryptRaider\CryptRaider\Binaries\Win64\UnrealEditor-CryptRaider.patch_0.exp
Successfully linked patch (0.000s)
Patch creation for module F:\Projects\Unreal\CryptRaider\CryptRaider\Binaries\Win64\UnrealEditor-CryptRaider.dll successful (0.000s)
File F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderBlueprint\Module.RiderBlueprint.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderBlueprint.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderBlueprint.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderGameControl\RiderGameControl.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderGameControl.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderGameControl.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderShaderInfo\RiderShaderInfo.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module F:\Projects\Unreal\CryptRaider\CryptRaider\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderShaderInfo.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderShaderInfo.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
---------- Finished (0.000s) ----------
```
It didn't affect my development process, but was pretty annoying. Also, there was a chance it could be a problem of my blueprints losing components (I guess, it is just livecoding bug).
So here what I've done:
1) I checked path `'C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink'` and there was no such directory. 
2) I manually created directory `'C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\'`
3) Now when I press live coding button, Unreal Engine recompiles all of my assets and after that crashes.
Last live coding log line:
`LogLiveCoding: Display:    Creating library C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RD.patch_0.lib and object C:\Users\{USER_NAME}\AppData\Local\Temp\UnrealLink\Femebav\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RD.patch_0.exp`
And indeed, now I have UnrealEditor-RD.patch_0 files in the directory.
Basically, now it successfully creates UnrealEditor-Rider***.patch_0 files and dies after executing them. 

4) If I delete the directory, then whole situation happens again. Everything works, but UnrealEditor-***.patch_0 files cannot be opened.

What else information can I provide to you to understand this bug?

**Comment by NullOne01** [comment]
It looks like, it is project specific bug. I've compiled another project and it works fine.

**Comment by cprehn** [comment]
Greetings - 

I am having the same issue. In my situation I have an existing project that I was using with Visual Studio. I installed and switched to Rider and the errors below occurred. It still compiles and runs. Not a blocking issue but a persistent error.

Like @NullOne01, if I create a new project. No errors show up in the console.

**Log:**

Registered process C:\Program Files\Epic Games\UE_5.0\Engine\Binaries\Win64\UnrealEditor.exe (PID: 2300)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RD.dll (0.927 MB)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderLink.dll (0.611 MB)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderLogging.dll (0.083 MB)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Binaries\Win64\UnrealEditor-UnrealLearningKit.dll (0.081 MB)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderBlueprint.dll (0.237 MB)
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderGameControl.dll (0.129 MB)
Ignoring compiland C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\MovingPlatform.cpp.obj because it is newer than the module it belongs to.
No PDB file found for module UnrealEditor-LiveCoding.dll. If this is a packaged build, make sure that debug files are being staged. Live coding will be disabled for this module.
Ignoring compiland C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\MovingPlatform.gen.cpp.obj because it is newer than the module it belongs to.
Ignoring compiland C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\UnrealLearningKit.init.gen.cpp.obj because it is newer than the module it belongs to.
Loading module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderShaderInfo.dll (0.055 MB)
Loaded 7 module(s) (0.000s, 4 translation units)
Live coding ready - Save changes and press Ctrl+Alt+F11 to re-compile code
Manual recompile triggered
---------- Creating patch ----------
Running C:\Program Files\Epic Games\UE_5.0\Engine\Binaries\DotNET\UnrealBuildTool\UnrealBuildTool.exe -Target="UnrealLearningKitEditor Win64 Development -Project=""C:/Users/Carl/code/UnrealProjects/UnrealLearningKit/UnrealLearningKit.uproject""" -LiveCoding -LiveCodingModules="C:/Program Files/Epic Games/UE_5.0/Engine/Intermediate/LiveCodingModules.txt" -LiveCodingManifest="C:/Program Files/Epic Games/UE_5.0/Engine/Intermediate/LiveCoding.json" -WaitMutex -LiveCodingLimit=100
Quick restart disabled when re-instancing is enabled.
  Log file: C:\Users\Carl\AppData\Local\UnrealBuildTool\Log.txt
  Target is up to date
  Total execution time: 1.14 seconds
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Lifetime.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\LifetimeDefinition.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\LifetimeImpl.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SequentialLifetimes.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SignalCookie.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\DateTime.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\IProtocol.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\IRdWireable.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\ISerializersOwner.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\IUnknownInstance.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdBindableBase.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdReactiveBase.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\WireBase.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\ExtWire.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdExtBase.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RName.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\InternRoot.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\InternScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Buffer.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Identities.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\MessageBroker.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Protocol.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdId.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\IScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SingleThreadSchedulerBase.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SimpleScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SingleThreadScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SynchronousScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\DefaultAbstractDeclaration.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\ISerializable.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Polymorphic.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdAny.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SerializationCtx.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\Serializers.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\hashing.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\thread_util.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\ByteBufferAsyncProcessor.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\PkgInputStream.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\PumpScheduler.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SocketWire.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\WireUtil.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RdTextBuffer.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\ActiveSocket.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\PassiveSocket.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\SimpleSocket.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\countdownlatch.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\async.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\cfg.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\color_sinks.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\file_sinks.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\fmt.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\spdlog.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\stdout_sinks.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\thirdparty.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RD\RD.cpp.obj was modified or is new
Building patch from 55 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RD.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RD.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderLink\Module.RiderLink.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderLink.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderLink.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderLogging\Module.RiderLogging.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderLogging.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderLogging.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\MovingPlatform.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\MovingPlatform.gen.cpp.obj was modified or is new
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Intermediate\Build\Win64\UnrealEditor\Development\UnrealLearningKit\UnrealLearningKit.init.gen.cpp.obj was modified or is new
Building patch from 3 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Binaries\Win64\UnrealEditor-UnrealLearningKit.dll
   Creating library C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Binaries\Win64\UnrealEditor-UnrealLearningKit.patch_0.lib and object C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Binaries\Win64\UnrealEditor-UnrealLearningKit.patch_0.exp
Successfully linked patch (0.000s)
Patch creation for module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Binaries\Win64\UnrealEditor-UnrealLearningKit.dll successful (0.000s)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderBlueprint\Module.RiderBlueprint.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderBlueprint.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderBlueprint.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderGameControl\RiderGameControl.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderGameControl.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderGameControl.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
File C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Intermediate\Build\Win64\UnrealEditor\Development\RiderShaderInfo\RiderShaderInfo.cpp.obj was modified or is new
Building patch from 1 file(s) for Live coding module C:\Users\Carl\code\UnrealProjects\UnrealLearningKit\Plugins\Developer\RiderLink\Binaries\Win64\UnrealEditor-RiderShaderInfo.dll
LINK : fatal error LNK1104: cannot open file 'C:\Users\Carl\AppData\Local\Temp\UnrealLink\Fijiqyd\HostProject\Plugins\RiderLink\Binaries\Win64\UnrealEditor-RiderShaderInfo.patch_0.exe'
Failed to link patch (0.000s) (Exit code: 0x450)
---------- Finished (0.000s) ----------

**Comment by NullOne01** [comment]
Somehow this bug disappeared. Here what I've done:
1) Invalidated cache in Rider by: Open Rider > File > Invalidate Caches.
2) Deleted "Binaries" folder in project folder. Unreal recompiled whole thing without a problem.

**Comment by rahulxiao** [comment]
I am facing some problem in live coding --

LOG:
---------- Creating patch ----------
Running D:\Game\UE_5.2\Engine\Build\BatchFiles\Build.bat -Target="HillEditor Win64 Development -Project=""G:/GAME PROJECTS/Hill/Hill.uproject""" -LiveCoding -LiveCodingModules="D:/Game/UE_5.2/Engine/Intermediate/LiveCodingModules.txt" -LiveCodingManifest="D:/Game/UE_5.2/Engine/Intermediate/LiveCoding.json" -WaitMutex -LiveCodingLimit=100
Quick restart disabled when re-instancing is enabled.
  Using bundled DotNet SDK version: 6.0.302
  Running UnrealBuildTool: dotnet "..\..\Engine\Binaries\DotNET\UnrealBuildTool\UnrealBuildTool.dll" -Target="HillEditor Win64 Development -Project=""G:/GAME PROJECTS/Hill/Hill.uproject""" -LiveCoding -LiveCodingModules="D:/Game/UE_5.2/Engine/Intermediate/LiveCodingModules.txt" -LiveCodingManifest="D:/Game/UE_5.2/Engine/Intermediate/LiveCoding.json" -WaitMutex -LiveCodingLimit=100
  Log file: C:\Users\rahul\AppData\Local\UnrealBuildTool\Log.txt
  Unhandled exception: System.IO.FileNotFoundException: Could not find file 'D:\Game\UE_5.2\Engine\Intermediate\LiveCodingModules.txt'.
  File name: 'D:\Game\UE_5.2\Engine\Intermediate\LiveCodingModules.txt'
     at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
     at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
     at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
     at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
     at System.IO.Strategies.FileStreamHelpers.ChooseStrategy(FileStream fileStream, String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
     at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
     at System.IO.File.InternalReadAllLines(String path, Encoding encoding)
     at System.IO.File.ReadAllLines(String path)
     at UnrealBuildTool.HotReload.PatchActionsForTarget(BuildConfiguration BuildConfiguration, TargetDescriptor TargetDescriptor, TargetMakefile Makefile, List`1 PrerequisiteActions, List`1 TargetActionsToExecute, Dictionary`2 InitialPatchedOldLocationToNewLocation, ILogger Logger) in D:\build\++UE5\Sync\Engine\Saved\CsTools\Engine\Source\Programs\UnrealBuildTool\System\HotReload.cs:line 565
     at UnrealBuildTool.BuildMode.Build(TargetMakefile[] Makefiles, List`1 TargetDescriptors, BuildConfiguration BuildConfiguration, BuildOptions Options, FileReference WriteOutdatedActionsFile, ILogger Logger) in D:\build\++UE5\Sync\Engine\Saved\CsTools\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 644
     at UnrealBuildTool.BuildMode.Execute(CommandLineArguments Arguments, ILogger Logger) in D:\build\++UE5\Sync\Engine\Saved\CsTools\Engine\Source\Programs\UnrealBuildTool\Modes\BuildMode.cs:line 248
     at UnrealBuildTool.UnrealBuildTool.Main(String[] ArgumentsArray) in D:\build\++UE5\Sync\Engine\Saved\CsTools\Engine\Source\Programs\UnrealBuildTool\UnrealBuildTool.cs:line 599
Build failed.
